### PR TITLE
fix: cannot build ml-api-adapter with current version of central-services-stream

### LINKED ISSUE #140: Cannot build ml-api-adapter with current version of central-services-stream
PR #133 changed Producer.isConnected() to require a parameter always.

This breaks the health check functionality in `ml-api-adapter`, where no parameter is passed to that function:
https://github.com/mojaloop/ml-api-adapter/blob/60aff84bcf234c4bab4fc2e982c84774c58e5fc2/src/lib/healthCheck/subServiceHealth.js#L49

### PR DESCRIPTION
**Pull request by kalinkrustev**
Fixes #140

### CODE REVIEW

File: .circleci/config.yml
Comment (kalinkrustev) [inline .circleci/config.yml:649]: `publish-prerelease` added above is an implementation for the [pre-release branches](https://docs.mojaloop.io/community/standards/creating-new-features.html#prerelease-branches) which I proposed recently.

This job will be useful to have in many other, so I am opening this issue https://github.com/mojaloop/ci-config/issues/38 to move the code to a central place.

**Comment by kalinkrustev** [comment]
@elnyry-sam-k I assigned you for review as this fix also includes an implementation for the pre-release branches.

There is also something strange, whereby Circle CI is reported as not signed the CLA, see this:
image

**Review by geka-evk — DISMISSED** [review]

File: .circleci/config.yml
Comment (elnyry-sam-k) [reply .circleci/config.yml:651]: Thanks @kalinkrustev , really appreciate your contribution here. 

There was no opposition on the DA and I had a discussion with James on this and we're ok to move forward with this. Thanks again!

**Comment by elnyry-sam-k** [comment]
> @elnyry-sam-k I assigned you for review as this fix also includes an implementation for the [pre-release branches](https://docs.mojaloop.io/community/standards/creating-new-features.html#prerelease-branches).
> 
> There is also something strange, whereby Circle CI is reported as not signed the CLA, see this: image

Hmm, that is technically true; there are commits on here shown from circleCI and apparently it hasn't signed, so looks okay. We can probably override that part (for example: https://github.com/mojaloop/central-services-stream/pull/141/commits/2eb3459cf0d43a9e0e8c28e0d74fdbc48b8bdb09 ).  But, let me look at it closer.

**Comment by elnyry-sam-k** [comment]
This repo also needs a codeowners file

**Comment by elnyry-sam-k** [comment]
These commits should show up from mojaloopci and not circleci, not sure where the diff is, need to check that.

**Comment by kalinkrustev** [comment]
> These commits should show up from mojaloopci and not circleci, not sure where the diff is, need to check that.

I fixed this here 28e139245986cce357304d3c09b7866030245fd9, the next commit 0edeeda4553f0954f076ad12b45ac0c6de3f3253 is from mojaloopci.

**Comment by kalinkrustev** [comment]
@elnyry-sam-k, please approve if all is good

**Comment by elnyry-sam-k** [comment]
perfect, thank you!

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**
